test/fixtures/video.txt

```diff
@@ -316,11 +381,6 @@ x
 .
 <p>@[(prezi]</p>
 .
-.
-@[toc]()
```

Comment:
Why is this being removed?

Reply:
This was a duplicate test. There is still an identical one on line 67.